util/SerializeMultiplayerCommon.cpp

```diff
@@ -143,9 +143,10 @@ template void MultiplayerLobbyData::serialize<freeorion_xml_iarchive>(freeorion_
 template <class Archive>
 void ChatHistoryEntity::serialize(Archive& ar, const unsigned int version)
 {
-    ar  & BOOST_SERIALIZATION_NVP(m_timestamp)
+    ar  & BOOST_SERIALIZATION_NVP(m_text)
```

Comment:
probably should add a version number and handle older versions when loading to preserve save compatibility